src/tokenizer.rs

```diff
@@ -102,6 +103,24 @@ pub enum Token {
     RBrace,
     /// Right Arrow `=>`
     RArrow,
+    /// Sharp `#` use for PostgreSQL Bitwise XOR operator
+    Sharp,
+    /// Tilde `~` use for PostgreSQL Bitwise NOT operator
+    Tilde,
+    /// Bitwise left operator `<<` use for PostgreSQL
+    ShiftLeft,
+    /// Bitwise right operator `>>` use for PostgreSQL
+    ShiftRight,
+    /// Exclamation Mark `!` use for PostgreSQL factorial operator
+    ExclamationMark,
+    /// Exclamation Mark `!!` use for PostgreSQL prefix factorial operator
+    DoubleExclamationMark,
+    /// Ampersat `@` use for PostgreSQL abs operator
+    Ampersat,
```

Comment:
It's commonly known as the "At sign". I had to Google it to make sure it wasn't a typo, and wikipedia says this term has not achieved wide use. Use `AtSign`?